README.md

```diff
@@ -1,44 +1,428 @@
-# test-mule
+# Test Mule
 
-[not ready for use]
+Test Mule is a library that allows you to use real browsers in your Jest tests. Test Mule is focused on helping you write tests that are [as similar as possible to how users use your application](https://twitter.com/kentcdodds/status/977018512689455106). It is built on [Puppeteer](https://github.com/puppeteer/puppeteer), [Testing Library](https://testing-library.com), and [jest-dom](https://github.com/testing-library/jest-dom).
+
+## Usage
+
+### Getting Started
+
+Test Mule integrates with Jest tests. If you haven't set up Jest yet, [here is Jest's getting started guide](https://jestjs.io/docs/en/getting-started).
+
+The [`withBrowser` wrapper](#withbrowser) tells Test Mule to launch a browser for the test. By default, a headless browser will be launched. The browser will close at the end of the test, unless the test failed. It is possible to have browser tests and non-browser tests in the same test suite.
+
+```js
+import { withBrowser } from 'test-mule';
+
+test(
+  'test name',
+  withBrowser(async () => {
+    // Your test code here
+  }),
+);
+```
+
+### Loading Content
+
+#### Option 1: Navigating to a real page
```

Comment:
Do we recommend this? If this isn't the preferred / expected way for people to use this tool, I recommend moving it off the front lines of our documentation and instead maybe just mention it's a supported alternative (and link to further documentation elsewhere).

Reply:
It is how people use Cypress. I think it is a valid use case.

Reply:
I'm not questioning it's validity at all. I'm just wondering if it's what we want to lead with. Listing it first in this way gives me the impression that it's the default / most-canonical method. Maybe you feel that's true? 

Part of why I think this is worth the bother if that this use case seems least representative of why TM is cool and special (IMO). If the reader stopped here, they'd be forgiven for thinking we'd sort of rebuilt Cypress, right? To me, the improved ergonomics are a **big** part of what's special about TM... do you agree? <-- this is the part I think is most interesting about this conversation.

Reply:
I reordered the 3 options. The "navigate to real page" option is now third.